src/main/scala/com/gu/identity/social/google/GoogleOpenIDConnect.scala

```diff
@@ -18,7 +18,7 @@ object GoogleOpenIDConnect {
     verifyToken(idToken, googleClientID)
 
   def verifyToken(idToken: IDToken, googleClientID: GoogleClientID): Either[AuthError, JWTUser] = {
-    val verifier = new GoogleIdTokenVerifier.Builder(new NetHttpTransport, JacksonFactory.getDefaultInstance)
+    val verifier = new GoogleIdTokenVerifier.Builder(new NetHttpTransport, GsonFactory.getDefaultInstance)
       .setAudience(List(googleClientID.id).asJavaCollection)
```

Comment:
I just merged the same change in identity and it seems to be fine. I think this implements the same interface as the old one